src/dev/impl/DevToys/Views/Tools/Generators/HashGenerator/HashGeneratorToolPage.xaml

```diff
@@ -42,6 +42,17 @@
                     <ComboBoxItem Tag="Base64" Content="{x:Bind ViewModel.Strings.OutputBase64}"/>
                 </ComboBox>
             </controls:ExpandableSettingControl>
+            <controls:ExpandableSettingControl
+                Title="HMAC Mode">
```

Comment:
Please globalize this string by adding a new entry in https://github.com/veler/DevToys/blob/main/src/dev/impl/DevToys/Strings/en-US/HashGenerator.resw . Then open https://github.com/veler/DevToys/blob/main/src/dev/impl/DevToys/LanguageManager.tt in Visual Studio, press Ctrl+S (even if there's no changes). This will re-regenerate https://github.com/veler/DevToys/blob/main/src/dev/impl/DevToys/LanguageManager.cs automatically and you will then be able to use the resource in the XAML.